hm_12/hm_012.py

```diff
@@ -0,0 +1,98 @@
+# task_num_1
+import json
+phone_book = {
+    "Alina Kravchenko": "0664686501",
+    "Egor Miroshnichenko": "0995678456",
+    "Ruslan Volikov": "0987656342",
+    "Nastya Lehenka": "0995676456"
+
+}
+
+json_phone_book = json.dumps(phone_book, indent=0)
+
+with open("json_phone_book.json", "x") as file:
```

Comment:
Your program can start only once. The second run raises error because of `x` modifier.

The whole point of this task is to keep your data in a file. Even when you close your program, the file should keep all previously entered data and in some way, be a database for your program. 

So, the first comment: load data from a file before you run the main loop and dump its content to a dict to work with it. 

Second comment: your file should keep all actual data that your phonebook has. In your case, you update the file only after the main loop ends, and for some reason, your user has only three attempts. And, if some error happens in the main loop, all unsaved data will be lost. It shouldn't work like this. I would suggest you update your phonebook right in places where your data is changing: `add` and `delete` commands. In this case, you will be 100% sure that all user-entered data is saved.